README.md

```diff
@@ -656,20 +748,30 @@ Thanks to the entire [SPKI WG](https://datatracker.ietf.org/wg/spki/about/) for
 
 We want to especially recognize [Mark Miller](https://github.com/erights) for his numerous contributions to the field of distributed auth, programming languages, and computer security writ large.
 
-# 9. FAQ
+# 12. FAQ
 
-## 9.1 What prevents an unauthorized party from using an intercepted UCAN?
+## 12.1 What prevents an unauthorized party from using an intercepted UCAN?
 
 UCANs always contain information about the sender and receiver. A UCAN is signed by the sender (the `iss` field DID) and can only be created by an agent in possession of the relevant private key. The recipient (the `aud` field DID) is required to check that the field matches their DID. These two checks together secure the certificate against use by an unauthorized party.
 
-## 9.2 What prevents replay attacks on the invocation use case?
+## 12.2 What prevents replay attacks on the invocation use case?
 
 A UCAN delegated for purposes of immediate invocation MUST be unique. If many requests are to be made in quick succession, a nonce can be used. The receiving agent (the one to perform the invocation) checks the hash of the UCAN against a local store of unexpired UCAN hashes.
 
 This is not a concern when simply delegating since presumably the recipient agent already has that UCAN.
 
-## 9.3 Is UCAN secure against person-in-the-middle attacks?
+## 12.3 Is UCAN secure against person-in-the-middle attacks?
 
 _UCAN does not have any special protection against person-in-the-middle (PITM) attacks._
 
 Were a PITM attack successfully performed on a UCAN delegation, the proof chain would contain the attacker's DID(s). It is possible to detect this scenario and revoke the relevant UCAN but does require special inspection of the topmost `iss` field to check if it is the expected DID. Therefore, it is strongly RECOMMENDED to only delegate UCANs to agents that are both trusted and authenticated and over secure channels.
+
+## 12.4 Why are the superuser and delegation abilities separate?
```

Comment:
Am I correct in interpreting this as `*` means all abilities including ones that may be introduced in the future. While `ucan/delegate` means all abilities that are present in proofs ? If so I think I'm starting to get the distinction, yet I think that could probably be signaled better.


In other words following capability

```
{ 
    "with": "owned://did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV/wnfs", 
    "can": "*" 
  }
 ```

Implies that all `wnfs://` capabilities of `did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV` had been delegated even though they aren't listed by `did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV`.

On the other hand following capability:

```js
{ 
    "with": "ucan:*", 
    "can": "ucan/DELEGATE" 
  }
```

implies that all the of the capabilities from all proofs have been (re)delegated.


Confusing bit here is that two either never overlap (in which case I think there's no point in having `ucan/delegate`) or following capability:

```js
{ 
    "with": "owned://did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV/wnfs", 
    "can": "ucan/DELEGATE" 
  }
``` 

has different semantics from the 1st one (with same `with` and  `"can": "*"`) and implies to select only explicitly delegated "wnfs" capabilities from "did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV" as opposed to all the "wnfs" capabilities from that actor.

If later is true I think disambiguating between these delegations using different with schemas seems a lot less confusing to me.

Reply:
Yup, that's the distinction! `ucan/delegate` only makes sense in the context of `ucan:`, in the same way that `msg/send` does make sense for email but not `wnfs:`.

I'll add some text to that effect 👍

Reply:
> Tell me about not having ucan/delegate then? msg/send also doesn't overlaps with CRUD actions, and often resources can use only one or the other.

There are few things that I think contribute to the confusion:

1. The `*` is a special form (only thing that does not need to be namespaced). It is used also in `with` to match anything. However in case of `can` it's semantics change to mean maximum rights to the resource.
2. It is confusing (and annoying) that `*` in `can` field can only be used with some protocol schemas and not the other. In turn it creates more opportunities to have invalid UCANs (I'm pretty sure we even have some of those in our system). It is true that not every `can` be used in conjunction with every `with`, but given a special form I'd expect it to.
3. Just like you could use `*` in `"with": "ucan:*"` to match pattern I would expect to do the same in `can: "store/*"` to match all the capabilities that were delegated namespaces with `store`.

In summary it feels composable but it is not and it is not intuitively obvious which things can be used in conjunction and which can't. This adds to complexity which I fear can lead to implementation incompatibilities due to misinterpretation.

It seems more intuitive to capture whether resource refers to explicitly delegated or implicitly owned capabilities in "resource" selector than in ability. I could be convinced that it should be captured in "ability", but even than using any other identifier but `*` would be far less confusing.


Alternatively encoding "superuser ability" in a different special form might worth considering

Reply:
Ok so here is an example of the delegated capabilities

```js
[
  { 
    "with": "owned://did:key:zAlice/wnfs", 
    "can": "*" 
  },
  {
     "with": "owned://did:key:zBob/wnfs",
     "can": "ucan/delegate"
  }
]
```

So here is what I find unclear about this:

1. What happens if Alice delegated just specific `wnfs` capabilities as opposed to super user ability ? Is above redelegation invalid or is it just redelegates ones that Alice delegated ?
2. What happens if Bob delegated "superuser ability" ? Does above UCAN redelegates all the `wnfs` abilities from Bob including future ones (which would mean it would be no different from using `"can": "*"`) or does not delegate any `wnfs` ability from Bob ? Or  perhaps it implies abilities that were known to "wnfs" at time the delegation occurred (I hope not this one) ?

Update: I am realizing now that `can: "*"` is reserved only for `owned:` schema, and  "ucan/delegate" for "ucan" schema which would mean above UCAN would be invalid, still I think it illustrates some of the confusion.

Reply:
@Gozala I just watched [@hugomrdias's UCAN talk from Iceland](https://www.youtube.com/watch?v=grec5KQeU2U) (which I missed because I was giving a concurrent talk at the same time), and the way he's framing the `foo/*` capabilities makes a ton of sense. It's a slightly different way of looking at the problem, but I think it's better. I'll make that change 👍